steamclock/steamclog-android
# Simplify Sentry setup (#84)

### Related Issue: #84 

### Summary of Problem:
* Some gradle configuration is not actually required and was causing the steps to setup Sentry support in our projects to be a little confusing

### Proposed Solution:
* Remove unrequired gradle settings (the sentry object should already be using those default values from the sound of it)

### Testing Steps:
* Run sample app
* Change log level to Release
* Send non-fatals
* Make sure non-fatals are being sent to Sentry and that the stack trace is readable

### Note
Readme will be updated to reflect this in #88
